README.md

```diff
@@ -197,3 +197,22 @@ To turn on the agent's internal debug logging:
 
 Note these logs are extremely verbose. Enable debug logging only when needed.
 Debug logging negatively impacts the performance of your application.
+
+## Roadmap to 1.0
+
+High-level roadmap as we work towards a 1.0 release:
+
+* Keep up with the specifications, including but not limited to:
+  * Implement all applicable semantic attributes
+    * Clearly document any additional attributes that are set
+        which are not defined by semantic conventions
+  * Standard metrics (still TBD, e.g.
+      [opentelemetry-specification#522](https://github.com/open-telemetry/opentelemetry-specification/issues/522))
+* Build out manual instrumentation for all existing auto-instrumentation
```

Comment:
Why this is needed for 1.0?

Reply:
Added:

> It is our goal to release 1.0 (GA) of the auto-instrumentation agent during the first wave of
OpenTelemetry 1.0 (GA) releases, along with as many manual instrumentation libraries as possible.

and removed the word `all` from this specific bullet point.